### DISCUSSION TITLE
The request timed out

### DISCUSSION THREAD

**Discussion by Skipperlla** [TheWidlarzGroup/react-native-video#3865] [Q&A]
### What are you trying to do?

I get the "request timed out" error (code -1001) for some of my users. How can I solve this?

![Screenshot 2024-05-31 at 16 56 05](https://github.com/TheWidlarzGroup/react-native-video/assets/68515357/ed20fc01-3398-4a4d-bda6-fece640ebc1a)

### Platforms

iOS

**Comment by KrzysztofMoch** [comment]
Hey @Skipperlla can you open ticket for it and provide more info like what platform, system version etc.

**Reply by Skipperlla** [reply]
iPhone 13

**Reply by MrXyfir** [reply]
Did you figure this out @Skipperlla? This has started happening randomly since updating Expo/RN/react-native-video.
